problems/lib/problem1.rb

```diff
@@ -0,0 +1,7 @@
+class Problem1
+  def sum_even_numbers(array)
+    sum = 0
+    array.each {|num| sum+=num if num.even?}

```

Comment:
You can guard against nil by raising an exception early, at the top of the method like this:
raise "Array must not be nil." if array.nil?
NB: I did's check for nil because the problems (all three in fact) state "Given an integer array of size N". A nil array cannot have a size, so I just assumed it is not.